### DISCUSSION TITLE
Problem installing el_core_web_sm

### DISCUSSION THREAD

**Discussion by stordopoulos** [explosion/spaCy#8979] [Help: Installation] [answered]
## How to reproduce the problem

I'm trying to install spaCy on Google's Colab (Google's version of Jupyter), and test its capabilities, as well as how well it can handle the Greek language. I had no problems installing:

- ```en_core_web_trf```
- ```el_core_news_sm```

But it seems that there's an issue install either of the Greek ```gsoc2018-spacy``` models:
- ```el_core_web_sm```
- ```el_core_web_lg ```

The code is pretty straight forward:

```
# Print CUDA version
!nvcc --version

# Install necessary packages for spaCy
!pip install -U pip setuptools wheel
# Install the latest version of spaCy
!pip install "spacy[cuda110,transformers,lookups]>=3.1"

# Install Greek models for spaCy
!python -m spacy download en_core_web_trf
!python -m spacy download el_core_web_lg
```

Here's a copy to the Colab Notebook: [Spacy_Greek_language.ipynb](https://colab.research.google.com/drive/1WzQS7p7yvBvFCI8hU_tCl5J5rS1UuCQl?usp=sharing)

```bash
2021-08-05 11:45:27.512036: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcudart.so.11.0

✘ No compatible package found for 'el_core_web_sm' (spaCy v3.1.1)

2021-08-05 11:45:31.568570: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library libcudart.so.11.0

✘ No compatible package found for 'el_core_web_lg' (spaCy v3.1.1)
```

## Your Environment
* Operating System: Linux-5.4.104+-x86_64-with-Ubuntu-18.04-bionic
* Python Version Used: 3.7.11
* spaCy Version Used: 3.1.1
* Environment Information: Build cuda_11.0_bu.TC445_37.28845127_0

**Comment by adrianeboyd** [comment, marked as answer]
Hi, the pretrained `el` models provided for the current version of spacy (v3.1.x) are all described here: https://spacy.io/models/el

The names are all with `news` instead of `web`. They might have had different names during internal development/testing, but as far as I can tell the models published by us were always named `el_core_news_*`.
